lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -434,3 +449,28 @@ def test_move_users_to_same_cohort(self):
             ],
             verify_order=False
         )
+
+
+@override_settings(MODULESTORE=TEST_DATA_XML_MODULESTORE)
+class TestResponsesReport(TestReportMixin, ModuleStoreTestCase):
+    """
+    Tests that CSV student responses report generation works.
+    """
+    def test_unicode(self):
+        course_key = CourseKey.from_string('edX/unicode_graded/2012_Fall')
+        self.course = get_course(course_key)
+        self.problem_location = Location("edX", "unicode_graded", "2012_Fall", "problem", "H1P1")
+
+        self.student = UserFactory(username=u'student\xec')
```

Comment:
Might be clearer to [add an encoding](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/tests/test_videos.py#L1) to the top of the file and then just [use the actual unicode characters](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/tests/test_videos.py#L93) here.